libraries/Microsoft.BotKit.Adapters.Slack/DialogData.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Microsoft.BotKit.Adapters.Slack
+{
+    /// <summary>
+    /// Data related to the dialog.
+    /// </summary>
+    public class DialogData
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DialogData"/> class.
+        /// </summary>
+        /// <param name="title">title for the Dialog.</param>
+        /// <param name="callback">callback id for the Dialog.</param>
+        /// <param name="submit">submit label for the Dialog.</param>
+        /// <param name="elements">List of elements for the Dialog.</param>
+        public DialogData(string title, string callback, string submit, List<DialogElement> elements)
```

Comment:
I would lean towards this being an empty constructor, and folks can use standard C# Init Syntax if they want to.

Reply:
Feedback applied. Thanks!